configure

```diff
@@ -159,7 +159,8 @@ Option		Description				Default
 --ssl=...	SSL library to use (gnutls, nss, openssl, auto)
 							$ssl
 --external_json_parser=0/1/auto	Use External JSON parser $external_json_parser
---systemd=0/1  Enable/disable systemd support $systemd
+--systemd=0/1	Enable/disable automatic detection	$systemd
```

Comment:
The $systemd should probably be on the next line?

Reply:
Probably yes, I have changed it.